README.md

```diff
@@ -1,14 +1,76 @@
 # rsmt2d
 
-Go implementation of two dimensional Reed-Solomon merkle tree data availability scheme.
+Go implementation of [two dimensional Reed-Solomon Merkle tree data availability scheme](https://arxiv.org/abs/1809.09044).
 
 [![GitHub Workflow Status](https://img.shields.io/github/workflow/status/lazyledger/rsmt2d/Tests)](https://github.com/lazyledger/rsmt2d/actions/workflows/ci.yml)
 [![Codecov](https://img.shields.io/codecov/c/github/lazyledger/rsmt2d)](https://app.codecov.io/gh/lazyledger/rsmt2d)
 [![GoDoc](https://godoc.org/github.com/lazyledger/rsmt2d?status.svg)](https://godoc.org/github.com/lazyledger/rsmt2d)
 
-Experimental software. Use at your own risk!
+## Example
 
-## Usage
+```go
+package main
+
+import (
+    "bytes"
+
+    "github.com/lazyledger/rsmt2d"
+)
+
+func main() {
+    // Size of each share, in bytes
+    bufferSize := 64
+    // Init new codec
+    codec := rsmt2d.NewLeoRSFF16Codec()
```

Comment:
I feel like the default example should not rely on cgo / leopard.

Reply:
Good point.

Reply:
Fixed in e500007 (that's a lot of zeroes).